src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar What is the point to try 200 MB/s when you say it's not sustainable for common use ?
Try again with the example given :P

Optimization was planned, but i go slowly or the user get afraid while seing 20 lines of code to test, like me with conflicts to solve :wink: and also it ask a recent FFmpeg so...
But for that i need to see the log to identify the hardware and adapt the FFmpeg optimisation.
Though i can give you the general use and you can adapt it to your computer:
`ffmpeg64.exe -y -hwaccel auto -i jellyfish-60-mbps-hd-hevc-10bit.mkv -c:v h264_dxva2 -tune fastdecode -preset superfast -crf 22 -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov -seekable 1 hevc_test1.mp4 -benchmark`
(or `h264_libfmx` for Intel).

I planned as well to ask user to test with the lastest FFmpeg version. Could you do that and compare with this ? Thank you.